doc/changelog/04-tactics/11760-firstorder-leaf.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  The default tactic used by :g:`firstorder` and :g:`intuition` is
+  :g:`auto with core` instead of :g:`auto with *`;
+  see :ref:`decisionprocedures` for details;
+  old behavior can be reset by using the `-compat 8.12` command-line flag;
```

Comment:
This is misleading.  There is no option to set the default solver for intuition, and the compat flag does not impact it.  I think it'd be nice if we added a flag for intuition like there is one for firstorder

Reply:
Indeed. This PR does not change the behavior of `intuition`. I’ve thus fixed the changelog.